CHANGELOG.md

```diff
@@ -24,7 +24,7 @@ Despite us having broken lockstep, we are continuing to release new versions of
 all Certbot components during releases for the time being, however, the only
 package with changes other than its version number was:
 
-*
+* Only depend on imgconverter extension for Sphinx >= 1.6
```

Comment:
Whoops. We should move this up to the fixed section. This is where we list the packages that changed in the release.

Reply:
oops. should be fixed now, I hope >.<